.rspec

```diff
@@ -1 +1 @@
---require spec_helper
+--require rails_helper
```

Comment:
Loading whole `rails_helper` for all specs might be an overkill but I guess we can back to it once we start seeing performance hit on tests for POROs 🙂